src/components/complete-profile/CompleteProfileBlock.tsx

```diff
@@ -42,6 +45,10 @@ const CompleteProfileBlock: FC<CompleteProfileBlockProps> = ({
   const homePage = useMatch(guestRoutes[userRole as UserRole].path)
   const [isOpen, setIsOpen] = useState(false)
 
+  const { openDrawer, closeDrawer, isOpen: isDrawerOpen } = useDrawer()
+  const [isOfferCreated, setIsOfferCreated] = useState(false)
+  const handleOpenDrawer = () => openDrawer()
```

Comment:
why did you create `handleOpenDrawer` function? You don' have to do it, and the `handleOpenDrawer` function will be recreated after each render, which can affect productivity. As far as I understand you want to rename the `openDrawer` function, so you could destructure with a renaming, like in [this example](https://stackoverflow.com/questions/57065348/destructuring-and-rename-property)

Reply:
I created this function following the example at the MyOffers.tsx components. But looks like you are right, I’ll try to remove it